code/drasil-printers/lib/Language/Drasil/TeX/Preamble.hs

```diff
@@ -84,6 +92,17 @@ addDef LessThan      = command2 "newcommand" "\\lt" "\\ensuremath <"
 addDef SetMathFont   = command "setmathfont" "Latin Modern Math"
 addDef SymbDescriptionP1 = command3 "newlist" "symbDescription" "description" "1"
 addDef SymbDescriptionP2 = command1o "setlist" (Just "symbDescription") "noitemsep, topsep=0pt, parsep=0pt, partopsep=0pt"
+addDef SaveBox       = command "newsavebox" "\\mybox"
+addDef ResizeExpr    = vcat [
+    command "newcommand" "\\resizeExpression" <> pure (sq "2") <> pure (text "{"),
+    command2 "savebox" "\\mybox" "$#1$",
+    command0 "ifdim" <> command0 "wd" <> command0 "mybox" <> pure (text ">#2") <> command0 "linewidth",
+    command3 "resizebox" "#2\\textwidth" "!" ("\\usebox" ++ brace "\\mybox"),
+    command0 "else",
+    command "usebox" "\\mybox",
+    command0 "fi",
+    pure (text "}")
```

Comment:
Do we have any helper functions for wrapping content in curly braces?

Reply:
There are functions to wrap items in curly braces; however, they all concatenate the items horizontally. Whereas in this particular instance, they need to be concatenated vertically. Here are some of the helper functions:

https://github.com/JacquesCarette/Drasil/blob/8ca26be6df6d3f4c24627e76d051637b0ff85b16/code/drasil-printers/lib/Language/Drasil/Printing/Helpers.hs#L62

https://github.com/JacquesCarette/Drasil/blob/8ca26be6df6d3f4c24627e76d051637b0ff85b16/code/drasil-printers/lib/Language/Drasil/Printing/Helpers.hs#L80

https://github.com/JacquesCarette/Drasil/blob/8ca26be6df6d3f4c24627e76d051637b0ff85b16/code/drasil-printers/lib/Language/Drasil/TeX/Helpers.hs#L28-L31

Reply:
The 'right' thing to do would be extract
```haskell
lbrack = text "{"
rbrack = text "}"
```
and use that in making the above code, i.e. both the new code and the places where it is currently 'hard coded'.

Reply:
Fixed in https://github.com/JacquesCarette/Drasil/pull/3910/commits/581da6258114fbbfaabcb8e68c906064b4e0e850. I could not find any other instances where this is present as they all make use of horizontal concat when wrapping items in `{}`.